src/main/java/Objects/Task.java

```diff
@@ -0,0 +1,46 @@
+package Objects;
+
+public class Task {
+    private String name;
+    private boolean mark = false;
+    public Task(String name,boolean mark){
```

Comment:
Should a new line be added between variable declaration and method declaration to improve readability?
```suggestion
    private boolean mark = false;
    
    public Task(String name,boolean mark){
```